### PR TITLE
Update css-shared.js demanded by issue-3027

### LINKED ISSUE #3027: (css) -webkit-font-smoothing: antialiased; are not highlight correctly
Trying to highlighting syntax for css line -webkit-font-smoothing: antialiased;
image

image

To reproduce:
https://codesandbox.io/s/wonderful-meninsky-vsj7y?file=/index.html:632-668

### PR DESCRIPTION
**Pull request by AndyKIron**
Added 'font-smoothing' attribute

Resolves #3027

### Checklist
- [ ] Added markup tests, or they don't apply here because...
- [x] Updated the changelog at `CHANGES.md`
- [ ] Added myself to `AUTHORS.txt`, under Contributors

### CODE REVIEW

**Comment by joshgoebel** [comment]
Awesome, please update CHANGES.md also.

File: CHANGES.md [hunk 1]
```diff
@@ -17,6 +17,7 @@ Language grammar improvements:
 - enh(php) Add support for Enums (#3004) [Ayesh][]
 - enh(ecmascript) Add built-in types [Vaibhav Chanana][]
 - enh(kotlin) Add `kts` as an alias for Kotlin (#3021) [Vaibhav Chanana][]
+- enh(css) Add `font-smoothing` to attributes list for CSS (#3027) [AndyKIron][]
```
Comment (joshgoebel) [inline CHANGES.md:20]: And you need to add the other half of the link on line ~35, etc...

Comment (AndyKIron) [reply CHANGES.md:20]: Thanks :)

**Review by joshgoebel — APPROVED** [review]

**Comment by joshgoebel** [comment]
Thanks for noticing and for the quick fix!

**Pull request merged**